README.Rmd

```diff
@@ -366,13 +197,28 @@ gc(verbose = FALSE)
 library(nycflights13)
 # rodbc does not support writing timestamps natively, so we remove that column
 flights2 <- as.data.frame(flights[, names(flights) != "time_hour"])
-rodbc <- dbConnect(RODBCDBI::ODBC(), dsn = "MicrosoftSQLServer", user = Sys.getenv("SQLSERVER_UID"), password = Sys.getenv("SQLSERVER_PWD"))
+
+rodbc <- 
```

Comment:
This also needs a timing for RODBC.

Reply:
[6d7f402](https://github.com/r-dbi/odbc/pull/652/commits/6d7f402ca1f1613021c4190f4608cd9aab926cb4). I'd guess that the previous version didn't have those because RODBC and RODBCDBI have the same back-end so they see similar timings, but that's worth calling out and demonstrating.